tests/test_defects.py

```diff
@@ -67,16 +67,42 @@ def setUp(self):
         self.flatExp = isrMock.FlatMock(config=mockImageConfig).run()
         (shapeY, shapeX) = self.flatExp.getDimensions()
 
-        # x, y, size tuples - NOTE: currently all are therefore square
+        # x, y, size tuples
         # always put edge defects at the start and change the value of nEdge
-        self.brightDefects = [(0, 15, 3), (100, 123, 1), (77, 90, 3)]
-        self.darkDefects = [(25, 0, 1), (33, 62, 2), (52, 21, 2)]
+
+        self.brightDefects = [(0, 15, 3, 3), (100, 123, 1, 1),
+                              (15, 1, 1, 50),
+                              (20, 1, 1, 25),
+                              (25, 1, 1, 8),
+                              (30, 1, 1, 2), (30, 5, 1, 3), (30, 11, 1, 5), (30, 19, 1, 5),
+                              (30, 27, 1, 4), (30, 34, 1, 15),
+                              (35, 1, 1, 2), (35, 5, 1, 3), (35, 11, 1, 2),
+                              (40, 1, 1, 2), (40, 5, 1, 3), (40, 19, 1, 5), (40, 27, 1, 4), (40, 34, 1, 15),
+                              (45, 10, 1, 2), (45, 30, 1, 3),
+                              [50, 10, 1, 1], [50, 12, 1, 1], [50, 14, 1, 1], [50, 16, 1, 1],
+                              [50, 18, 1, 1], [50, 20, 1, 1], [50, 22, 1, 1], [50, 24, 1, 1],
+                              [50, 26, 1, 1], [50, 28, 1, 1], [50, 30, 1, 1], [50, 32, 1, 1],
+                              [50, 34, 1, 1], [50, 36, 1, 1], [50, 38, 1, 1], [50, 40, 1, 1],
+                              [55, 20, 1, 1], [55, 22, 1, 1], [55, 24, 1, 1], [55, 26, 1, 1],
+                              [55, 28, 1, 1], [55, 30, 1, 1],
+                              (60, 1, 1, 18), (60, 20, 1, 10), (61, 2, 2, 2), (61, 6, 2, 8),
+                              (70, 1, 1, 18), (70, 20, 1, 10), (68, 2, 2, 2), (68, 6, 2, 8),
+                              (75, 1, 1, 18), (75, 20, 1, 10), (73, 2, 2, 2), (73, 6, 2, 8), (76, 2, 2, 2),
+                              (76, 6, 2, 8),
+                              (80, 1, 1, 18), (80, 20, 1, 10), (81, 2, 2, 2), (81, 8, 2, 8),
+                              (87, 1, 1, 18), (87, 20, 1, 10), (85, 2, 2, 2), (85, 8, 2, 8),
+                              (93, 1, 1, 12), (93, 15, 1, 20), (91, 2, 2, 2), (91, 7, 2, 2),
+                              (94, 2, 2, 2), (94, 7, 2, 2),
+                              (91, 18, 2, 3), (91, 24, 2, 3), (94, 18, 2, 3), (94, 24, 2, 3)]
```

Comment:
I think it would be clearer if each specific column test had exactly its the list of defects that it was testing defined in the test. So, The `(15,1,1,50)` column shouldn't be here, just in that one test. It would make it easier to see what's going on. And then you could note that the `50` length is because of splitting along the mock amp boundary.